### PR TITLE
Development

### PR DESCRIPTION
**Pull request by KaushikShivam**

### CODE REVIEW

**Comment by Ispirett** [comment]
Wow!!! really great work on this project folks it looks pro :1st_place_medal:  There are just some minor issues to take care of before its complete.
- [x] On line 74 does not match anything so it can be deleted.
- [x]  The **fieldset** tag is used to group related set of data in this context <fieldset> it would group email phone and password. But not individually like you did. Instead, you can just use a div tag to wrap the  content

File: index.html [hunk 1]
```diff
@@ -7,6 +7,81 @@
...
+                </li>
+                <li>
+                    <a href="">turbotax</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section>
```
Comment (Ispirett) [inline index.html]: Sections should always have a  **heading**  tag example h1 or h4. you hide the heading but its good practice to have for optimizing SEO. If not you can just use a div tag.

Comment (KaushikShivam) [reply index.html]: Hello, Ispirett
Thanks a lot for the kind compliments and suggestions. We have made the necessary changes.

**Comment by Oghenebrume50** [comment]
Hi team,  great work so far really super. Just like Ispirett suggested, your section should have a heading tag and also use `border-style: none;`  and ` outline-style: none` to remove that grey border from your buttons. 
No need to resubmit for review after doing this, merge to master and move ahead at once. Kudos

**Pull request merged**